src/Core/Collections/Leaderboard/Leaderboard.tsx

```diff
@@ -56,14 +65,27 @@ const Leaderboard = (): ReactElement => {
     leaderboardTimeRange === LeaderboardTimeRange.IGBO_VOICE_ATHON &&
     !moment().isBetween(igboVoiceathonStartDate, igboVoiceathonEndDate);
 
-  const handleUpdateLeaderboard = (event) => {
-    const updatedLeaderboard = event.target.value as LeaderboardType;
-    if (leaderboard !== updatedLeaderboard) {
-      setLeaderboard(updatedLeaderboard);
+  // const handleUpdateLeaderboard = (event) => {
+  //   const updatedLeaderboard = event.target.value as LeaderboardType;
+  //   if (leaderboard !== updatedLeaderboard) {
+  //     setLeaderboard(updatedLeaderboard);
+  //   }
+  // };
+  const handleLeaderboardClick = (index: number) => () => {
```

Comment:
instead of passing in the index of the leaderboard, pass in the `leaderboard.uid` which should be the type `LeaderboardType`

```suggestion
  const handleLeaderboardClick = (leaderboardId: LeaderboardType) => () => {
```

Reply:
Reason for not passing a uid is covered here https://github.com/nkowaokwu/igbo-api-admin/pull/438#discussion_r1287448199 will store the leaderboard option config instead.